pytorch_lightning/trainer/trainer.py

```diff
@@ -723,21 +723,21 @@ def _run(self, model: LightningModule) -> Optional[Union[_EVALUATE_OUTPUT, _PRED
                                 |                             ||
                         create accelerator                    ||
                                 |                             ||
-                         {self.dispatch}                      ||
+                         {self._dispatch}                     ||
                                 |                             ||  LIGHTNING
                   {self.accelerator.start_training}           ||
                 or {self.accelerator.start_evaluating}        ||
                 or {self.accelerator.start_predicting}        ||  FLOW
                                 |                             ||
                          {self.run_stage}                     ||
                                 |                             ||  DIRECTION
-                        {self.run_train}                      ||
-                     or {self.run_evaluation}                 ||
-                     or {self.run_predict}                    ||
+                        {self._run_train}                     ||
+                     or {self.run_evaluation}                ||
```

Comment:
why leave out `run_evaluation`?

Reply:
initially i hadn't updated this because TrainLoop calls `run_evaluation` too, but I've updated it now to also be private